src/components/about-us/section-05/list.js

```diff
@@ -0,0 +1,364 @@
+  `}
+`
+
+const Acomplishments = styled.div `
+  width: 100%;
+  ${screen.mobile`
+    transform-style: preserve-3d;
+    margin-top: 2px;
+    margin-bottom: ${props => props.unfold ? '9px' : '0'};  
+  `}
+`
+
+const MonthLabel = styled.div `
+  display: block;
+  background: ${colors.white};
+  width: 54px;
+  height: 23px;
+  padding: 5px 0;
+  text-align: center;
+  p{
+    font-size: 12px;
+    font-weight: bold;
+  }
+  ${screen.overDesktop`
+    p{
+      font-size: 14px;
+    }
+  `}
+  ${screen.tabletAbove`
+    display: ${props => props.monthOrder === 0 ? 'block' : 'none'};
+  `}  
+`
+
+const MonthlyAcomplishments = styled.div `
+  margin-bottom: 2px;
+  background: ${colors.gray.gray96};
+  min-height: 76px;
+`
+
+const Acomplishment = styled.div `
+  width: 100%;
+  margin: 0;
+  color: ${colors.black};
+  ${screen.mobile`
+    position: relative;
+    min-height: 76px;
+    height: ${props => props.unfold ? 'auto' : '76px'};
+    transition: ease ${transitionDuration} all;
+    border-bottom: solid 1px ${colors.white};
+    background: ${colors.gray.gray96};
+    &:nth-child(odd) {
+      transform-origin: 50% 0 0;
+      transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(-90deg)'};
+      &:before {
+        content: '';
+        background-image: linear-gradient(to top, ${colors.gray.gray64}, ${colors.gray.gray96} 80%);
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: ${props => props.unfold ? '0' : '100%'};
+        z-index: calc(${defaultZIndex} - 1);
+        transition: height .25s;      
+      }
+    }
+    &:nth-child(even) {
+      transform-origin: 50% 100% 0;
+      transform: perspective(300px) ${props => props.unfold ? 'rotateX(0deg)' : 'rotateX(90deg)'};
+      margin-top: ${props => props.unfold ? '0' : 'calc(-76px * 2)'};
+      &:before {
+        content: '';
+        background-image: linear-gradient(to bottom, ${colors.gray.gray64}, ${colors.gray.gray96} 80%);
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: ${props => props.unfold ? '0' : '100%'};
+        z-index: calc(${defaultZIndex} - 1);
+        transition: height .25s;      
+      }
+    }
+    &:last-child {
+      display: ${props => !props.unfold && props.isOdd ? 'none' : 'block'};
+    }  
+  `}
+`
+
+export default class List extends PureComponent {
+  constructor(props) {
+    super(props)
+    this.yearContent = []
+  }
+  _getMonth = (month) => {
+    switch(month) {
+      case 1:  
+        return '一月'
+      case 2:
+        return '二月'
+      case 3:
+        return '三月'
+      case 4:
+        return '四月'
+      case 5:
+        return '五月'
+      case 6:
+        return '六月'
+      case 7:
+        return '七月'
+      case 8:
+        return '八月'
+      case 9:
+        return '九月'
+      case 10:
+        return '十月'
+      case 11:
+        return '十一月'
+      case 12:
+        return '十二月'
```

Comment:
Since we want to adopt i18n, these month strings should be defined in other constant file.